Monika After Story/game/script-story-events.rpy

```diff
@@ -191,34 +181,49 @@ label mas_gender_redo_neither_same:
     m 1eka "But just know that it doesn't matter to me..."
     return
 
-label mas_gender_trans:
-    if persistent._mas_pm_is_trans:
-        $ menu_question = "And what gender do you identify as?"
-    else:
-        $ menu_question = "Oh, okay! {w=0.3}And what gender do you identify as?"
 
-    m 3eub "[menu_question]{nw}"
-    $ _history_list.pop()
-    menu:
-        m "[menu_question]{fast}"
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="mas_transgender_reveal",
+            category=["you"],
+            prompt=(
+                "I want to tell you that I am transgender"
+                if persistent._mas_pm_is_trans else
+                "I want to tell you that I am no longer transgender"
+            ),
+            unlocked=False,
+            pool=True,
+            rules={"no_unlock": None},
+        ),
+        markSeen=True
+    )
 
-        "Male":
-            $ persistent.gender = "M"
+label mas_transgender_reveal:
+    if persistent._mas_pm_is_trans:
+        m 1eua "TODO: YOU ARE TELLING ME THAT YOU ARE NO LONGER TRANSGENDER"
```

Comment:
This isn’t finished, all it really does is serve as a base or guideline for a full-fledged topic.

```suggestion
    if persistent.mas_pm_genderfluid = False
            m 1wdu "You’re no longer transgender?"
            m “...”
            m “Why are you detransitioning, [player]?”
            $ _history_list.pop()
            menu:
                m “Why are you detransitioning, [player]?”
                
                ”I’m gender fluid.”:
                    $ persistent.mas_pm_genderfluid = “True”
                    m “Oh, you are?”
                    m “I’m so happy that 
                
                “I don’t feel like a [boy] anymore.”:
                    m “I see...”
                    
                ”I was exploring which gender I was most comfortable with.”:
                    m “Ah, I understand.”
                    m “I hope you managed to learn something new about yourself.”
                    
                “I wasn’t accepted being transgender.”:
                    m “[player]!”
                    m “I...”
                    m “[player], I’m so sorry...”
                    m “You should be accepted for whoever you choose to be.”
                    m “...”
                    m “Are you sure you’re going to detransition, [player]...?”
                    $ _history_list.pop()
                    menu:
                        m “Are you sure you’re going to detransition, [player]...?”
                        “Yes.”:
                            m “...”
                            m “Okay, [player].”       
                            
                        ”No.”:
                            m “I’m glad...”
                            m “I hope you can give yourself more time to think about this.”
                            m “Please don’t let others keep you from your happiness.”
                
                    m “Remember that I’ll always love and accept you no matter what, okay?”
                    
                ”I don’t want to talk about it.”:
                    m “Alright, [player]...”
                    m “Just know I’ll always accept you for who you are, okay?”
                    m “You’re everything to me~”
                
            else:
            m “You’re detransitioning again?”
            m “Alright, [player].”
            m “I’m happy you get to express yourself this way~”
```
If we go this route, there should be a **default persistent._mas_pm_genderfluid = False** so there will be different routes based on whether the player has seen the topic before or not. This is a draft, and I’m not transgender myself. If anyone can think of a better way to word things in here, please let me know. The last thing I want is for this topic to come across as transphobic. And it probably will need a few persistent updates based on what the player chooses.